js/setup.js

```diff
@@ -1,196 +1,159 @@
 'use strict';
 
-var NAME_PATTERN = [
-  'Иван',
-  'Хуан Себастьян',
-  'Мария',
-  'Кристоф',
-  'Виктор',
-  'Юлия',
-  'Люпита',
-  'Вашингтон'
-];
-var SURNAME_PATTERN = [
-  'да Марья',
-  'Верон',
-  'Мирабелла',
-  'Вальц',
-  'Онопко',
-  'Топольницкая',
-  'Нионго',
-  'Ирвинг'
-];
-var COAT_COLOR_PATTERN = [
-  'rgb(101, 137, 164)',
-  'rgb(241, 43, 107)',
-  'rgb(146, 100, 161)',
-  'rgb(56, 159, 117)',
-  'rgb(215, 210, 55)',
-  'rgb(0, 0, 0)',
-];
-var EYES_COLOR_PATTERN = [
-  'black',
-  'red',
-  'blue',
-  'yellow',
-  'green',
-];
-var FIREBALL_COLOR_PATTERN = [
-  '#ee4830',
-  '#30a8ee',
-  '#5ce6c0',
-  '#e848d5',
-  '#e6e848',
-];
-
-var WIZARDS_QUANTITY = 4;
-
-var getRandomInt = function (min, max) {
-  return Math.floor(Math.random() * (max - min + 1)) + min;
-};
-
-var getRandomArrValue = function (patternName) {
-  return patternName[getRandomInt(0, patternName.length - 1)];
-};
-
-var getRandomWizards = function () {
-  var wizards = [];
-
-  for (var i = 0; i < WIZARDS_QUANTITY; i++) {
-    var newWizard = {
-      name: getRandomArrValue(NAME_PATTERN) + ' ' + getRandomArrValue(SURNAME_PATTERN),
-      coatColor: getRandomArrValue(COAT_COLOR_PATTERN),
-      eyesColor: getRandomArrValue(EYES_COLOR_PATTERN),
-    };
-
-    wizards.push(newWizard);
-  }
-
-  return wizards;
-};
-
-var renderWizard = function (wizard) {
-  var newWizard = similarWizardTemplate.cloneNode(true);
-
-  newWizard.querySelector('.setup-similar-label').textContent = wizard.name;
-  newWizard.querySelector('.wizard-coat').style.fill = wizard.coatColor;
-  newWizard.querySelector('.wizard-eyes').style.fill = wizard.eyesColor;
-
-  return newWizard;
-};
-
-var renderWizardsFragment = function (wizards) {
-  var fragment = document.createDocumentFragment();
-
-  for (var i = 0; i < wizards.length; i++) {
-    fragment.appendChild(renderWizard(wizards[i]));
-  }
-
-  return fragment;
-};
-
-var openSetup = function (evt) {
-  evt.preventDefault();
-  setupElement.classList.remove('hidden');
+window.setup = (function () {
+  var setupElement = document.querySelector('.setup');
+  var setupDefaultX;
+  var setupDefaultY;
```

Comment:
Режет глаз, переменная для дефолтного значения, а его нету, задай значения
```suggestion
  var setupDefaultX = 0;
  var setupDefaultY = 0;
```